addons/mail/static/src/core/common/thread_model.js

```diff
@@ -53,9 +53,12 @@ export class Thread {
     description;
     /** @type {Set<import("@mail/core/common/follower_model").Follower>} */
     followers = new Set();
+    selfFollower = undefined;
+    followersCount = undefined;
     isAdmin = false;
     loadOlder = false;
     loadNewer = false;
+    loadMoreFollowers = false;
```

Comment:
I have 2 issue with this: 

* The name is confusing, it feel like a function name, probably because there is also a function named the same way (could be renamed `shouldLoadMoreFollowers`?).
* This is an UI concern, I don't see why this is inside the thread model. I feel like this should be in the state of the component follower list?

Reply:
I've renamed field to `followersFullyLoaded`.
This should stay in models because this is fully dependent on model and service.
This could have been a getter (it can be deduced from `followersCount`, `followers`, and `selfFollower`), but we make a dedicated field because it's more CPU efficient to determine from the fetch (`mail/thread/data` and `mail/thread/follower`)

Reply:
In the end I turned the field into getter, it's not really costly